peerconnection.go

```diff
@@ -1706,7 +1754,13 @@ func (pc *PeerConnection) AddTrack(track TrackLocal) (*RTPSender, error) {
 	pc.mu.Lock()
 	defer pc.mu.Unlock()
 	for _, t := range pc.rtpTransceivers {
-		if !t.stopped && t.kind == track.Kind() && t.Sender() == nil {
+		currentDirection := t.getCurrentDirection()
+		// According to https://www.w3.org/TR/webrtc/#dom-rtcpeerconnection-addtrack, if the
+		// transceiver can be reused only if it's currentDirection never be sendrecv or sendonly.
+		// But that will cause sdp inflate in sfu. So we only check currentDirection's current value,
```

Comment:
Maybe remove reference to the work `sfu`? Just mention that without re-use of transceivers, SDP will inflate for every track add/removal.

Reply:
removed. just thinking only sfu will have much track remove/add